packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -119,3 +119,9 @@
 .total-price {
   @apply font-semibold overflow-hidden text-ellipsis text-lg whitespace-nowrap;
 }
+
+.main-content {
+  @apply grid grid-cols-[calc(100vw_-_1.5rem)] grid-rows-[min-content] gap-x-4 my-16 min-h-[calc(100vh_-_4rem)];
+  @apply tablet:grid-cols-[minmax(0,500px)_minmax(18rem,1fr)] tablet:gap-x-8 tablet:mt-[81px] tablet:ml-3 tablet:mr-3 tablet:m-auto;
```

Comment:
Should this be `tablet:grid-cols-[minmax(359px,500px)_20rem]`? This would allow the first column's width to range between 359px and 500px, while the second column would be kept fixed at 320px (20rem vs 18rem (288px)).

`tablet:mt-[80px] tablet:mx-3 tablet:mb-auto`

- In Header, we updated it from 81px to 80px.
- `tablet:mx-3` will combine `tablet:ml-3 tablet:mr-3`
- Forgot the 'b' in 'mb' for margin-bottom :)

Reply:
Ah good catches thank you. I've made the recommended changes.

Regarding the `tablet:grid-cols-*` I'd like to keep it like this below. (I changed the 18rem to 20rem)
- `tablet:grid-cols-[minmax(0px,500px)_minmax(20rem,1fr)]`

I realize this changes the logic a little, but doing it this way, it makes the break from `desktop` to `tablet` a little less harsh. It keeps the second column the same size it was before, and then shrinks it to `20rem`. 

I'd also like to get rid of the 359px, so that we can get rid of this arbitrary value that doesn't actually get used in any case. (The default `grid-cols-[calc(100vw_-_1.5rem)]` kicks in before the 1st column reaches 0px)

Reply:
If `grid-cols-[calc(100vw_-_1.5rem)]` kicks in before the 1st column reaches 0px, then should 0px be updated to `min-content` instead? -> `tablet:grid-cols-[minmax(min-content,500px)_minmax(20rem,1fr)]`?

Reply:
Ah sweet. Yup that makes sense.